nucypher/network/server.py

```diff
@@ -6,33 +6,38 @@
 import kademlia
 from apistar import http, Route, App
 from apistar.http import Response
-from bytestring_splitter import VariableLengthBytestring, BytestringSplitter
+from constant_sorrow import constants
 from kademlia.crawling import NodeSpiderCrawl
 from kademlia.network import Server
 from kademlia.utils import digest
-from umbral import pre
-from umbral.fragments import KFrag
 
+from bytestring_splitter import VariableLengthBytestring, BytestringSplitter
 from nucypher.config.configs import NetworkConfiguration
+from nucypher.crypto.constants import PUBLIC_ADDRESS_LENGTH, PUBLIC_KEY_LENGTH
 from nucypher.crypto.kits import UmbralMessageKit
 from nucypher.crypto.powers import EncryptingPower, SigningPower
 from nucypher.keystore.threading import ThreadedSession
-from nucypher.network.protocols import NucypherSeedOnlyProtocol, NucypherHashProtocol, \
-    dht_with_hrac_splitter, InterfaceInfo
+from nucypher.network.protocols import NucypherSeedOnlyProtocol, NucypherHashProtocol, InterfaceInfo
 from nucypher.network.storage import SeedOnlyStorage
+from umbral import pre
+from umbral.fragments import KFrag
 from umbral.keys import UmbralPublicKey
 from umbral.signing import Signature
-from nucypher.crypto.constants import PUBLIC_ADDRESS_LENGTH, PUBLIC_KEY_LENGTH
 
 
 class NucypherDHTServer(Server):
     protocol_class = NucypherHashProtocol
     capabilities = ()
     digests_set = 0
 
-    def __init__(self, id=None, *args, **kwargs):
+    def __init__(self, node_storage, treasure_map_storaage, id=None, *args, **kwargs):
```

Comment:
`storaage`? Why double `a`?

Reply:
Typo.  It was non-trivial to fix it in a rebase, so I fixed it in #345.

Reply:
To be clear - we're merging a typo that will be fixed in #345?